src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_Format.txt

```diff
@@ -0,0 +1,200 @@
+﻿>> Text(DateTime(2022,12,1,23,45,59), "YYYY-MM-DD HH:MM:SS AM/PM")
+"2022-12-01 11:45:59 PM"
+
+>> Text(DateTime(2022,12,1,23,45,59), "yYyY-Mm-dd Hh:Mm:Ss Am/pM")
+"2022-12-01 11:45:59 PM"
+
+>> Text(DateTime(2022,12,1,23,45,59), "YYYY-MM-DD HH:MM:SS A/P")
+"2022-12-01 11:45:59 P"
+
+>> Text(DateTime(2022,12,1,23,45,59), "YYYY-MM-DD HH:MM:SS a/pa/p")
```

Comment:
>a/pa/p [](http://example.com/codeflow?start=59&length=6)

It's nice that this is working in C# - it doesn't work in Power Apps, we'll need to fix it there.